src/test/java/com/google/opensesame/mockObjects/MockGHObjects.java

```diff
@@ -0,0 +1,109 @@
+package com.google.opensesame.mockObjects;
+
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.ArrayList;
+import java.util.List;
+import org.kohsuke.github.GHRepository;
+import org.kohsuke.github.GHUser;
+
+public class MockGHObjects {
```

Comment:
Let's add a documentation comment, it would be helpful to mention that this class has 2 static methods to mock a GithubUser and GithubRepository.